glean/metrics.yaml

```diff
@@ -949,6 +949,11 @@ sample:
       transport:
         description: The Host network type (i.e wifi or 4g)
         type: string
+      operation_type:
+        description: |
+          The type of operation the controller was performing. Ex. none,
+          silent-switch, switch
```

Comment:
Can't we extract this same information from: https://dictionary.telemetry.mozilla.org/apps/mozilla_vpn/metrics/sample_controller_step?

What exactly is it we want to answer with this new collection?

Reply:
We want to collect errors, right? Some errors are generated when we are in StateOn, which could come from StateSwitching.

Reply:
Sure, but we can infer the state at the time the error occured by looking at the other event I linked to there.

Reply:
Yes, but it's not easy to compute I guess. The handshake timeout could happen in Connecting or Confirming state and we then need to go back in time to see the sequence of State changes per ping. Is it doable? Is it better to keep the current PR?

Reply:
It is doable. It's not hard, it's medium difficulty. It requires some SQL or Looker calculations, but we should be able to do that. I am not very much in favour of adding redundant collection.